app/Models/Schedule.php

```diff
@@ -103,7 +103,7 @@ class Schedule extends Model
      */
     public static $validationRules = [
         'server_id' => 'required|exists:servers,id',
-        'name' => 'nullable|string|max:255',
+        'name' => 'nullable|string|max:75',
```

Comment:
This change will break all of the schedules that already have a longer name until someone manually edits it to have a shorter name.

Reply:
Changed back to 255